.gitmodules

```diff
@@ -26,3 +26,6 @@
 	path = include/quickcpplib/byte
 	url = https://github.com/martinmoene/byte-lite.git
 	branch = master
+[submodule "include/quickcpplib/span-lite"]
+	path = include/quickcpplib/span-lite
+	url = https://github.com/martinmoene/span-lite
```

Comment:
```suggestion
```
This seems to be redundant with the other span-lite submodule.

Reply:
done, a small git mishap !